java/java.platform/test/unit/src/org/netbeans/modules/java/platform/queries/PlatformSourceForBinaryQueryTest.java

```diff
@@ -82,6 +80,23 @@ public void testUnregisteredPlatform() throws Exception {
         assertNull(result);
     }
 
+    public void testUnregisteredJDK11Platform() throws Exception {
+        File wd = getWorkDir();
```

Comment:
[PlatformSourceForBinaryQueryTest](https://github.com/apache/netbeans/actions/runs/3804935671/jobs/6473338798#step:20:579) seems to pass OK.